recipes/fast-cdr/all/conanfile.py

```diff
@@ -1,4 +1,4 @@
-from conan import ConanFile
+from conan import ConanFile, conan_version
```

Comment:
This is not required, if you revert this then the PR can be automatically merged 

It's going to be removed eventually so no point adding checks